docs/install.md

```diff
@@ -37,13 +37,24 @@ Python
 pipx install sherlock-project
 ```
 
-### Build python package from source (useful for contributors)
+### Build live package from source (useful for contributors)
+
+Building an editable (or live) package links the entry point to your current directory, rather than to the standard install location. This is often useful when working with the code base, as changes are reflected immediately without reinstallation.
+
+Note that the version number will be 0.0.0 for pipx local builds unless manually changed in the pyproject file (it will prompt the user for an update).
 
 ```bash
-# pipx is recommended, but pip may suffice if pipx is unavailable
-git clone https://github.com/sherlock-project/sherlock.git
-cd sherlock
-pipx install .
+# Assumes repository cloned, and pwd is repository root
+pipx install -e .
+```
+
+### Run package from source (no install)
+
+If you'd rather not install directly to your system, you can import the module at runtime with `-m`.
+
+```bash
+# Assumes repository cloned, and pwd is repository root
+python3 -m sherlock user123 user789
```

Comment:
`-m` is not needed. `python sherlock user123 user789` also works

Reply:
That changes with #2111 due to more-proper packaging with Poetry. Leads to import errors and such.

I may be able to address that, but it's not the typical way packages are ran :man_shrugging:

Essentially, when ran as a script, there isn't a proper namespace, and imports have to be relative to the local. When packaged, there is a namespace, and imports have to be self-package-based. I'm explaining this terribly but it's early/late